espanol/canudos.md

```diff
@@ -185,4 +185,6 @@ Al despertar Resines te pregunta si eres [Batman](batman/Batman.md)
 
 A lo que respondes: "Yo solo vine aquí a hablar de mi [libro](libro/libro.md)".
 
-Luego de su viaje extraordinario se acordó de su [máquina del tiempo](maquina/tiempo/maquina_del_tiempo.md).
\ No newline at end of file
+Luego de su viaje extraordinario se acordó de su [máquina del tiempo](maquina/tiempo/maquina_del_tiempo.md).
+
+[En su viaje al pasado pudo comocer el poder de los dioses, quienes le enseñaron como invocar a Pegaso](pegaso/pegaso.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.